applications/nrf5340_audio/src/bluetooth/bt_stream/le_audio.c

```diff
@@ -34,3 +34,39 @@ bool le_audio_ep_state_check(struct bt_bap_ep *ep, enum bt_bap_ep_state state)
 
 	return false;
 }
+
+int le_audio_freq_get_hz(const struct bt_audio_codec_cfg *codec)
+{
+	int freq_id;
+	int freq_hz;
+
+	freq_id = bt_audio_codec_cfg_get_freq(codec);
+	if (freq_id < 0) {
+		return freq_id;
+	}
+
+	freq_hz = (bt_audio_codec_cfg_freq_to_freq_hz(freq_id));
+	if (freq_hz < 0) {
+		LOG_WRN("Invalid codec frequency detected, %d", freq_hz);
+	}
+
+	return freq_hz;
+}
+
+int le_audio_duration_us(const struct bt_audio_codec_cfg *codec)
+{
+	int frame_dur;
+	int dur_us;
+
+	frame_dur = bt_audio_codec_cfg_get_frame_dur(codec);
+	if (frame_dur < 0) {
+		return frame_dur;
+	}
+
+	dur_us = (bt_audio_codec_cfg_frame_dur_to_frame_dur_us(frame_dur));
+	if (dur_us < 0) {
+		LOG_WRN("Invalid codec duration detected, %d", dur_us);
```

Comment:
IMO, all these prints should be LOG_ERR

Reply:
If I change this to an assert then it traps the error, would this be acceptable?